### DISCUSSION TITLE
Still no benefits school education pack

### DISCUSSION THREAD

**Discussion by prosir** [community/community#139098] [GitHub Education]
### Hi everyone,

Hello,

I requested sunday for my github benefits packs for school education. But still not got any acces to the benefits. Any idea how long it does take ? I want to start with my projects haha :)

Greatings from the Netherlands!

Jacco

**Comment by davevad93** [comment]
Hi @prosir ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @prosir was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
